lib/matplotlib/rcsetup.py

```diff
@@ -93,6 +94,21 @@ def __call__(self, s):
         raise ValueError(msg)
 
 
+def _single_string_color_list(s, scalar_validator):
```

Comment:
```suggestion
def _single_string_color_list(s, scalar_validator):
    """
    Convert *s* to a list of colors interpreting it either a color sequence name,
    or a string containing single-letter colors.
    """
```
A docstring helps to understand what this does.